openml-lightgbm-provider/pom.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ The copyright of this file belongs to Feedzai. The file cannot be
+  ~ reproduced in whole or in part, stored in a retrieval system,
+  ~ transmitted in any form, or by any means electronic, mechanical,
+  ~ photocopying, or otherwise, without the prior permission of the owner.
+  ~
+  ~ (c) 2020 Feedzai, Strictly Confidential
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>openml-java</artifactId>
+        <groupId>com.feedzai</groupId>
+        <version>0.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>openml-lightgbm-provider</artifactId>
+
+    <name>Feedzai OpenML LGBM Provider</name>
+    <description>Contains an example for the implementation of a LGBM Machine Learning Model and CLassifier
+    </description>
+
+    <properties>
+        <openml.version>1.0.3</openml.version>
+        <lightgbmlib.version>2.3.150</lightgbmlib.version>
+    </properties>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>com.feedzai</groupId>
+            <artifactId>openml-api</artifactId>
+            <version>${openml.version}</version>
```

Comment:
remove the version and define the scope to provided

`<scope>provided</scope>`

Reply:
similar to the other dependencies that have `<version>${openml.version}</version>`